webscrapbook/cli.py

```diff
@@ -282,6 +299,28 @@ def main():
         help="""generate more assistant files. (with --book)""")
     parser_config.add_argument('-e', '--edit', default=False, action='store_true',
         help="""edit the config file. (with --book or --user)""")
+    
+    # subcommand: scrapbook
+    parser_scrapbook = subparsers.add_parser('scrapbook', aliases=['scrap'],
+        help=cmd_scrapbook.__doc__, description=cmd_scrapbook.__doc__)
+    parser_scrapbook.set_defaults(func=cmd_scrapbook)
+    parser_scrapbook.add_argument('directory',
+        help="""scrapbook directory""")
+    subparser_scrapbook = parser_scrapbook.add_subparsers(metavar='SCRAPBOOK_COMMAND',
+        help="""The scrapbook command to run. Add --help(-h) after the command for usage details.""")
+    parser_sort = subparser_scrapbook.add_parser('sort')
+    parser_sort.set_defaults(func=cmd_sort)
+    parser_sort.add_argument('folder',
+        help="""id for folder to sort""")
+    parser_sort.add_argument('key',
+        choices=['title','create', 'modify', 'source', 'comment'],
+        help=""""metadata key to sort on""")
+    parser_sort.add_argument('-dir', '--direction',
+        choices=['a','d'],
+        default='a',
+        help="""direction of sort ascending or descending""")
```

Comment:
Default value should be included in the help message.

Reply:
I'll fix that.

Reply:
Fixed!